prediction_market_agent/agents/microchain_agent/balance_functions.py

```diff
@@ -0,0 +1,36 @@
+from microchain import Function
+from prediction_market_agent_tooling.tools.balances import get_balances
+from web3 import Web3
+
+from prediction_market_agent.utils import APIKeys
+
+
+class GetOtherWalletXDAIBalance(Function):
+    @property
+    def description(self) -> str:
+        return f"Use this function to fetch xDai and wxDai balance of someone else given his wallet address."
+
+    @property
+    def example_args(self) -> list[str]:
+        return ["0xSomeAddress"]
+
+    def __call__(self, wallet_address: str) -> str:
+        balances = get_balances(Web3.to_checksum_address(wallet_address))
+        return f"Balance of {wallet_address} is {balances.xdai} xDai and {balances.wxdai} wxDai."
```

Comment:
Should we also display `sDAI` balance?

Reply:
I returned wxDai as well becaues `get_balances` is returning it (not sDai). Do you think these agents should be aware of sDai?

Reply:
Probably not for right now (since it requires sDAI function on PMAT). If you could create an issue and link here, that's good enough for me

Reply:
@coderabbitai Please create an issue for adding the ability to get the balance of sDai for a general agent. With a consideration to maybe get balance of arbitrary token?